### ISSUE TITLE
Capslock light not working. Configuration disappeared (only once) under Mojave.

### ISSUE THREAD

**Issue by LucaMarconato** [tekezo/Karabiner-Elements#1570]
Today I updated to macOS Mojave and now Karabiner Elements does not work anymore with my internal keyboard. I have no external keyboard to try it.

If I try to add a new modification or complex modification, after quitting and reopening Karabiner Elements, the modification are not fond anymore.

I am not sure, but I think that right after installing Mojave Karabiner Elements was still working.
Then, all of the sudden (or at least I do not know what the cause could have been), I noticed that the Caps Lock light stopped working.
After a while the light started working again (maybe after awaking from sleep) but now Karabiner Elements is broken. I will try to reboot now.

**Comment by LucaMarconato** [comment]
Rebooting Karabiner Elements started working again and the modifications now are kept when quitting the program. Anyway the caps lock light does not work again. 

If I close Karabiner Elements it starts working again.

If I reopen Karabiner Elements the modifications keep working, but the not the light anymore.

**Comment by LucaMarconato** [comment]
I am using the beta version 12.1.5, I updated from the previous stable version when I experienced the problems that I have listed in my first post

**Comment by a-triant** [comment]
#1502

**Comment by a-triant** [comment]
Duplicate of #1502

**Comment by a-triant** [comment]
About non working LED light is duplicate of #1502
